### PR TITLE
New Keyboard: KPRepublic BM60 RGB

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

This is another port into QMK firmware from kbfirmware.com. This is a 60% with per key RGB leds. Customer who asked me to do this tested it and confirmed it working as I don't have an actual unit. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/bm60hsrgb/config.h [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
```
Comment (noroadsleft) [inline keyboards/bm60hsrgb/config.h]: I've been suggesting new Product IDs within `VENDOR_ID 0xFEED`, to try to get these unique.

```suggestion
#define PRODUCT_ID      0xEF8C
```

**Comment by mechmerlin** [comment]
I've renamed the directory and associated files to bm60rgb.

File: keyboards/bm60rgb/keymaps/default/readme.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# The default keymap for bm60hsrgb
```
Comment (noroadsleft) [inline keyboards/bm60rgb/keymaps/default/readme.md]:
```suggestion
# The default keymap for bm60rgb
```

**Comment by noroadsleft** [comment]
Do we want to put this in a `kprepublic` vendor folder, ahead of #5206?

**Comment by mechmerlin** [comment]
ooof that's an old PR. I think we should just merge this in for the time being. That PR has merge conflicts, so it can be moved as part of the merge conflict resolution.

**Comment by noroadsleft** [comment]
Thanks!

**Comment by ADDUB2** [comment]
I'm having some issues with this pcb the underglow got turned off the moment I flashed it and all the commands for lighting I flashed onto it don't work.

**Comment by daniarnaizg** [comment]
Underglow doesn't work. You cannot control it once you have flashed it.

**Comment by jrodan** [comment]
any idea how to fix it?
is that a problem with the pcb itself that you can not control underglow by itself?

**Pull request merged**